assets/js/theme/common/carousel/utils/arrowAriaLabling.js

```diff
@@ -1,26 +1,28 @@
-const NUMBER = '[NUMBER]';
-const integerRegExp = /[0-9]+/;
-const lastIntegerRegExp = /(\d+)(?!.*\d)/;
+export const updateTextWithLiveData = (textForChange, firstData, secondData) => {
+    const NUMBER = '[NUMBER]';
+    const integerRegExp = /[0-9]+/;
+    const lastIntegerRegExp = /(\d+)(?!.*\d)/;
+    const valueToReplace = textForChange.includes(NUMBER) ? NUMBER : integerRegExp;
+
+    return textForChange
+        .replace(valueToReplace, firstData)
+        .replace(lastIntegerRegExp, secondData);
+};
```

Comment:
also I think as now we using this code as a util we can create a separate file for it

Reply:
done